### ISSUE TITLE
Activation as 'N' for SEDenseNetImageNet121

### ISSUE THREAD

**Issue by ialhashim** [titu1994/keras-squeeze-excite-network#10]
I got activation must be one of "softmax" or "sigmoid" for 121

https://github.com/titu1994/keras-squeeze-excite-network/blob/07fc3132e9cbcb1b3d95c6a98bbdf55ab8be6ccd/se_densenet.py#L184

**Comment by titu1994** [comment]
Thanks for pointing that out. I'll fix it soon.

**Comment by titu1994** [comment]
This has been patched.
